src/Solr/Reindex/Handlers/SolrReindexImmediateHandler.php

```diff
@@ -100,7 +100,7 @@ protected function processGroup(
         $logger->info("Running '$cmd'");
 
         // Execute script via shell
-        $process = new Process($cmd);
+        $process = new Process([$cmd]);
```

Comment:
The [Symphony doc](https://symfony.com/doc/current/components/process.html#usage) says to pass the parameters as array entries:
```php
$process = new Process(['ls', '-lsa']);
```

So it probably needs to be something like this:
```suggestion
        $process = new Process([
            $scriptTask, 
            "index={$indexClassEscaped}",
            "class={$class}",
            "group={$group}",
            "groups={$groups}",
            "variantstate={$statevar}",
            "verbose=1"
        ]);
```

Reply:
Yes agree that would be better, have updated